Bottom: dd2a7c3959e28966b59d46e83be8ee3831b3badd
Top:    057edc20976415995ffe1c7466b372eee5307111
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-22 07:23:05 -0300

Refresh of wip-manual-document-mt-safety-locale.patch

---

diff --git a/manual/locale.texi b/manual/locale.texi
index 7d04c64..fb6de4e 100644
--- a/manual/locale.texi
+++ b/manual/locale.texi
@@ -243,7 +243,7 @@ The symbols in this section are defined in the header file @file{locale.h}.
 @c  _nl_find_locale
 @c   getenv LC_ALL and LANG envromt
 @c   _nl_load_locale_from_archive
-@c   _nl_expand_alias
+@c   _nl_expand_alias asmalloc, selfdeadlock, memleak, fdleak, lockleak
 @c    libc_lock_lock selfdeadlock, lockleak
 @c    bsearch ok
 @c     alias_compare ok
@@ -266,7 +266,7 @@ The symbols in this section are defined in the header file @file{locale.h}.
 @c   _nl_load_locale
 @c   __gconv_compare_alias
 @c   strndup asmalloc, memleak
-@c   strcasecmp_l (C locale)
+@c   strcasecmp_l ok (C locale)
 The function @code{setlocale} sets the current locale for category
 @var{category} to @var{locale}.  A list of all the locales the system
 provides can be created by running
